srcCpp/RTIDDSImpl.cxx

```diff
@@ -671,37 +683,35 @@ bool RTIDDSImpl<T>::ParseConfig(int argc, char *argv[])
                 _useUnbounded = MAX_BOUNDED_SEQ_SIZE;
             }
             _isLargeData = true;
-        } else if (_scan_max_size <= (unsigned long)(std::min)(MAX_SYNCHRONOUS_SIZE,MAX_BOUNDED_SEQ_SIZE)) {
+        } else {
```

Comment:
No need for a 'if' statement here. We are all ready taking in a count all the range of values with 'else'